packages/manager/src/managers/telemetry/types.ts

```diff
@@ -210,6 +210,7 @@ type FieldAddedSegmentEvent = SegmentEvent<
 		contentType: "page type" | "custom type" | "slice";
 		allowText?: boolean;
 		repeat?: boolean;
+		variants?: string[];
```

Comment:
❓ Interesting, so we can send an array directly to Segment / Amplitude? How does it's translated in Amplitude, as a string?

Reply:
irrc, the page builder team has some weird types for tracking events, only allowing strings

maybe it's because of the proxy we use, idk

but both segment and amplitude allow strings, numbers, arrays, booleans... which could improve a lot our analytics, imo